app/Http/Controllers/CompetitionsController.php

```diff
@@ -37,7 +47,10 @@ public function create()
      */
     public function store(Request $request)
     {
-        //
+        $this->validate($request, $this->validation_rules);
+        $input = $request->all();
+        Competition::create($input);
+        return redirect('/competitions');
```

Comment:
It's probably a good idea to use named routes here: `redirect()->route('competitions.index')`